examples/hcp-aks-demo/main.tf

```diff
@@ -148,7 +168,7 @@ resource "azurerm_network_security_rule" "ingress" {
   access                      = "Allow"
   protocol                    = "Tcp"
   source_port_range           = "*"
-  destination_port_range      = "80"
+  destination_port_range      = "8080"
```

Comment:
In consul-dataplane, a non-root envoy ingress gateway process cannot bind it's listener to a privileged port 80 (<1024).